inc/admin-ui.php

```diff
@@ -287,3 +287,22 @@ function render_posts_column( $column, $post_id ): void {
 	}
 }
 add_action( 'manage_profile_posts_custom_column', __NAMESPACE__ . '\render_posts_column', 10, 2 );
+
+/**
+ * Remove the author dropdown from the post editor.
+ * Done by removing author support.
+ */
+function remove_author_dropdown() {
+	// Allow filtering of post types to remove author dropdown.
+	$post_types = apply_filters( 'byline_manager_remove_author_dropdown', [] );
```

Comment:
Let's default this to Byline Manager's supported post types, so that for any post type where we're using Byline Manager, the author box is hidden by default. Users could still use this filter to customize the list of post types where author support is removed, but it would have a default that would be applicable to most sites.

Also, let's rename this filter to `byline_manager_remove_author_support` instead, which is a little more clear about what it does.